www/widgets/lab2-browser.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
```

Comment:
These files are all the same - use a pandoc template?

Reply:
That's a good idea!

Reply:
It templatized it, but we don't have an easy way to link chapter numbers (like `4`) to chapter names (like `html`) so all the templates right now point to the "graphics" chapter. Not ideal, I'll think about ways to fix it.